setup.py

```diff
@@ -52,6 +52,7 @@
             'eea.volto.policy',
             'pas.plugins.oidc',
             'collective.taxonomy',
+            'collective.volto.subsites==1.3.1',
```

Comment:
This pin is not good for Plone 6 usage

Reply:
Done in [c7b653a](https://github.com/eea/eea.kitkat/pull/19/commits/c7b653a42b6a0237af1e27f0a48578ea19e39afa)